### PR TITLE
Creating  blog page and some minor view and config fixes

### PR DESCRIPTION
**Pull request by vitorrgg**

### CODE REVIEW

**Comment by vitorrgg** [comment]
Pending the creation of diferent default posts.

File: template/views/partials/organisms/blog-grid.ejs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<section class="items-carousel my-4">
+  <div class="card-deck">
+    <div class="row align-items-center">
+    <% for (var post in posts)  {%>
```
Comment (leomp12) [inline template/views/partials/organisms/blog-grid.ejs]: `let` is better here:
```js
<% for (let post in posts) { %>
```
indentation is wrong also

File: template/views/partials/organisms/blog-grid.ejs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      <div class="col-lg-4 col-md-6 mb-5">
+        <div class="card" style="" href=" <%= posts[post] %>">
```
Comment (leomp12) [inline template/views/partials/organisms/blog-grid.ejs]: style="" is empty, so you should remove the attribute

Comment (leomp12) [reply template/views/partials/organisms/blog-grid.ejs]: wrong indentation again

Comment (leomp12) [reply template/views/partials/organisms/blog-grid.ejs]:
```ejs
        <div class="card" href="<%= posts[post] %>">
```

File: template/views/partials/organisms/blog-grid.ejs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+          <a href="<%= post %>"><img src="<%= posts[post].thumbnail %>" class="card-img-top" alt="..."></a>
```
Comment (leomp12) [inline template/views/partials/organisms/blog-grid.ejs]: why alt="..."?

Comment (leomp12) [reply template/views/partials/organisms/blog-grid.ejs]: you should also break this line of code:
```ejs
<a href="<%= post %>">
  <img src="<%= posts[post].thumbnail %>" class="card-img-top" alt="...">
</a>
```

File: template/views/partials/organisms/blog-grid.ejs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+          <div class="card-body">
+            <a href="<%= post %>"><h5 class="card-title"><%= posts[post].title %></h5></a>
```
Comment (leomp12) [inline template/views/partials/organisms/blog-grid.ejs]: Break this line of code....

File: template/views/partials/organisms/blog-grid.ejs [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+            <p class="card-text"><%= posts[post].description %></p>
+            <a href=" <%= post %>" class="btn btn-primary">Ler mais</a>
```
Comment (leomp12) [inline template/views/partials/organisms/blog-grid.ejs]:
```ejs
<a href="<%= post %>" class="btn btn-primary">
  Ler mais
</a>
```

File: template/views/partials/organisms/blog-grid.ejs [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<section class="items-carousel my-4">
+  <div class="card-deck">
+    <div class="row align-items-center">
+    <% for (var post in posts)  {%>
+      <div class="col-lg-4 col-md-6 mb-5">
+        <div class="card" style="" href=" <%= posts[post] %>">
+          <a href="<%= post %>"><img src="<%= posts[post].thumbnail %>" class="card-img-top" alt="..."></a>
+          <div class="card-body">
+            <a href="<%= post %>"><h5 class="card-title"><%= posts[post].title %></h5></a>
+            <p class="card-text"><%= posts[post].description %></p>
```
Comment (leomp12) [inline template/views/partials/organisms/blog-grid.ejs]:
```ejs
<p class="card-text">
  <%= posts[post].description %>
</p>
```

**Review by leomp12 — CHANGES_REQUESTED** [review]

**Review by leomp12 — APPROVED** [review]

**Pull request merged**